### ISSUE TITLE
How to get USB camera working?

### ISSUE THREAD

**Issue by dumtux** [YuzukiHD/TinyVision#37]
I'd like to connect and use an USB webcam to capture images.
How can I include `uvcvideo` package in image build and make the USB webcam automatically detected when it's connected?

**Comment by dumtux** [comment]
I've added these lines to [config-4.9](https://github.com/YuzukiHD/Yuzukilizard/blob/master/Software/BSP/device/config/chips/v851s/configs/lizard/linux-4.9/config-4.9)

```
CONFIG_VIDEOBUF2_CORE=y
CONFIG_VIDEOBUF2_MEMOPS=y
CONFIG_VIDEOBUF2_VMALLOC=y 
CONFIG_MEDIA_USB_SUPPORT=y
CONFIG_USB_VIDEO_CLASS=y
CONFIG_USB_GSPCA=y
```

And I also installed `fswebcam` package and `usbutils` package by running `make menuconfig`.
For installing `usbutils`, due to confliction between `eudev` and `libudev-zero`,
I have to uninstall `libinput` package and then uninstall `eudev` .

I've built the image using `lunch` and `mp` command.

After booting the board with the new image, I still cannot find `fswebcam` and `lsusb` commands in the shell.
And USB camera driver is still not working.

![image](https://github.com/user-attachments/assets/6410d010-3562-438a-a30a-0031ac2fdca8)
